src/nublado/controller/services/builder/lab.py

```diff
@@ -854,25 +933,53 @@ def _build_pod_init_containers(
                 image_pull_policy=spec.image.pull_policy.value,
                 resources=resources.to_kubernetes(),
                 security_context=as_root if spec.privileged else as_user,
-                volume_mounts=mounts,
+                volume_mounts=c_mounts,
             )
             containers.append(container)
 
+        # If it's a "science" type site, we run the "set up a landing page"
+        # container before startup.
+        #
+        # We want all the mounts because we are copying tutorials material from
+        # a shared, read-only filesystem into the user home directory.
+        if self._config.env.get("RSP_SITE_TYPE", "") == "science":
```

Comment:
Just a thought, but the behaviour of this env variable spawning an extra container might not be immediately obvious when looking at the config schema. Also is there ever a case where we would want to allow sites to opt out regardless of  `RSP_SITE_TYPE` ? 

So the thought is whether to add `standardLandingpage: true` similar to `standardInithome`, or do you think that's not really needed?